packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.tsx

```diff
@@ -214,4 +217,11 @@ export class ScrollablePane extends BaseComponent<IScrollablePaneProps, {}> {
       container.appendChild(sticky.content);
     });
   }
+
+  private _findScrollablePaneParent(element: HTMLElement): HTMLElement {
+    while (element.parentElement !== this.refs.root && element.parentElement !== null) {
+      element = element.parentElement;
```

Comment:
having a difficult time understanding what this change does!

what exactly are you trying to do? What is findScrollablePaneParent doing?